### PR TITLE
Explicitly set vm image to avoid using ubuntu16

### PR DESCRIPTION
**Pull request by praveenkuttappan**
VM image is not set for prepare pipeline and devops uses implicit image name ubuntu16. Job is failing due to incorrect image name when ubuntu16 is used by pipeline. This PR is to set image name to windows 2019

### CODE REVIEW

**Comment by benbp** [comment]
I know it's going to be a pain to update all the repo references, but since you're changing it to `jobs` it probably doesn't belong in the `steps` directory anymore.

File: eng/common/pipelines/templates/steps/prepare-pipelines.yml [hunk 1]
```diff
@@ -13,125 +13,128 @@ parameters:
   - name: TestsConventionOptions
     type: string
     default: ''
+jobs:
+- job: PreparePipelines
+  pool:
+    vmImage: 'windows-2019'
```
Comment (benbp) [inline eng/common/pipelines/templates/steps/prepare-pipelines.yml:19]: Why not ubuntu 20?

Comment (praveenkuttappan) [reply eng/common/pipelines/templates/steps/prepare-pipelines.yml:19]: I have used same image name as in other generate pipeline.

**Comment by praveenkuttappan** [comment]
> I know it's going to be a pain to update all the repo references, but since you're changing it to `jobs` it probably doesn't belong in the `steps` directory anymore.

I think this will need to be done as multiple steps. First add new template under `jobs` without purging current one, update each language reference to new job template and then purge common `steps` template.

**Comment by praveenkuttappan** [comment]
I have found that this change is not required and image name is set directly on devops for this. Closing this PR since it is not required anymore.

**Pull request closed without merging**